### PR TITLE
feat(#6): Adds configuration parameters to database services

### LINKED ISSUE #6: Improvement in SQL services
Currently, we are not configuring SQL services with a user and password, we should not only detect which kind of database but also the configuration file where this information is placed, and use it to configure the service.

### PR DESCRIPTION
**Pull request by lordofthejars**
Fixes #6

### CODE REVIEW

**Review by rohanKanojia — APPROVED** [review]

File: detectors/src/main/java/io/jshift/odo/detectors/DatabaseConfigurationExtractorLoader.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import java.util.stream.StreamSupport;
+
+public class DatabaseConfigurationExtractorLoader {
+
+
+    private static ServiceLoader<DatabaseConfigurationExtractor> loader = ServiceLoader.load(DatabaseConfigurationExtractor.class);
+
+    private List<DatabaseConfigurationExtractor> databaseConfigurationExtractors = new ArrayList<>();
+
+    public List<DatabaseConfigurationExtractor> detectors(Extractor extractor) {
```
Comment (devang-gaur) [inline detectors/src/main/java/io/jshift/odo/detectors/DatabaseConfigurationExtractorLoader.java]: shouldn't this method be called extractors ?

Comment (devang-gaur) [reply detectors/src/main/java/io/jshift/odo/detectors/DatabaseConfigurationExtractorLoader.java]: or ConfigExtractor ?

**Review by devang-gaur — CHANGES_REQUESTED** [review]
just a tiny change requested.  everything else looks perfect.

File: detectors/src/main/java/io/jshift/odo/detectors/DatabaseConfigurationExtractorLoader.java (hunk 1 above)
Comment (lordofthejars) [reply detectors/src/main/java/io/jshift/odo/detectors/DatabaseConfigurationExtractorLoader.java]: true.

**Pull request merged**